NOTICE.txt

```diff
@@ -171,6 +171,21 @@ COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
 IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+--------------------------------------------------------------------
+Dependency: github.com/coreos/go-semver
```

Comment:
Is this package still somewhere used?

Reply:
It shouldn't be any more. This line probably got re-introduced during a rebase or something. I will remove it. Thanks for catching!